Calculator/CalculatorBrain.swift

```diff
@@ -18,14 +21,61 @@ class CalculatorBrain {
     var operatorType: OperatorType?
     
     func addOperandDigit(_ digit: String) -> String {
-        
+        guard operatorType == nil else {
+            operand2String.append(digit)
+            return operand2String
+        }
+        operand1String.append(digit)
+        return operand1String
```

Comment:
This is an interesting way to add digits to the operands. It looks fine and does work. Generally swift doesn't like to compare values against nil. It looks like the 'guard' is what allows this to not throw an error. I'm interested in how you came to this conclusion.